application/frontend/controllers/ResetController.php

```diff
@@ -109,7 +109,10 @@ public function actionCreate()
                 'status' => 'error',
                 'error' => 'user not found',
             ]);
-            return new \stdClass();
+            throw new NotFoundHttpException(
+            \Yii::t('app', 'User not found'),
```

Comment:
Looks like we lost a bit of indentation here.
```suggestion
                \Yii::t('app', 'User not found'),
```